### PR TITLE
Fixed docker-compose for basic case

### LINKED ISSUE #1734: ValueError: Container 05efd8b1a0 does not have a com.docker.compose.container-number label
Running docker compose with a LEMP server yml file, caused the following error:
```
 	Pulling vic.mysql (mysql:latest)...
latest: Pulling from library/mysql
Digest: sha256:0d8465f6e13e7f56680e47a24159e8ebbe93c1c3c679e15bb44511e7bfcac5a9
Status: Downloaded newer image for library/mysql:latest
Creating vic.mysql
Pulling vic.php (php:7.0-fpm)...
7.0-fpm: Pulling from library/php
Digest: sha256:ac5cfcef5ba3874cbc68d17b159e48ab76170171ddd8f6e960e9918ced55e4f7
Status: Downloaded newer image for library/php:7.0-fpm
Traceback (most recent call last):
  File "/usr/bin/docker-compose", line 9, in <module>
    load_entry_point('docker-compose==1.5.2', 'console_scripts', 'docker-compose')()
  File "/usr/lib/python2.7/dist-packages/compose/cli/main.py", line 54, in main
    command.sys_dispatch()
  File "/usr/lib/python2.7/dist-packages/compose/cli/docopt_command.py", line 23, in sys_dispatch
    self.dispatch(sys.argv[1:], None)
  File "/usr/lib/python2.7/dist-packages/compose/cli/docopt_command.py", line 26, in dispatch
    self.perform_command(*self.parse(argv, global_options))
  File "/usr/lib/python2.7/dist-packages/compose/cli/main.py", line 171, in perform_command
    handler(project, command_options)
  File "/usr/lib/python2.7/dist-packages/compose/cli/main.py", line 546, in up
    detached=detached
  File "/usr/lib/python2.7/dist-packages/compose/project.py", line 264, in up
    detached=detached
  File "/usr/lib/python2.7/dist-packages/compose/service.py", line 389, in execute_convergence_plan
    container = self.create_container(do_build=do_build)
  File "/usr/lib/python2.7/dist-packages/compose/service.py", line 292, in create_container
    number or self._next_container_number(one_off=one_off),
  File "/usr/lib/python2.7/dist-packages/compose/service.py", line 523, in _next_container_number
    numbers = [c.number for c in containers]
  File "/usr/lib/python2.7/dist-packages/compose/container.py", line 87, in number
    self.short_id, LABEL_CONTAINER_NUMBER))
ValueError: Container 05efd8b1a0 does not have a com.docker.compose.container-number label
```

Attaching logs as well.
[VCH-0-3949-container-logs.tar.gz](https://github.com/vmware/vic/files/389506/VCH-0-3949-container-logs.tar.gz)

[robot-logs.zip](https://github.com/vmware/vic/files/389510/robot-logs.zip)

### PR DESCRIPTION
**Pull request by sflxn**
Cache label from docker create.  Docker-compose relies on it.

Fixes #1734

### CODE REVIEW

**Comment by dougm** [comment]
lgtm

**Comment by cgtexmex** [comment]
Is the thought we'll just store this locally for now and then tackle persistence of labels later?

File: tests/test-cases/Group3-Docker-Compose/3-3-Docker-Compose-Basic.robot [hunk 1]
```diff
@@ -1,28 +1,25 @@
...
-    #${rc2}  ${output}=  Run And Return Rc And Output  DOCKER_HOST=${vch-ip}:2375 docker-compose --file basic-compose.yml logs
-    #Log  ${output}
-    #${rc2}  ${output}=  Run And Return Rc And Output  DOCKER_HOST=${vch-ip}:2375 docker-compose --file basic-compose.yml stop
-    #Log  ${output}
-    #Should Be Equal As Integers  ${rc}  0
-    #Should Be Equal As Integers  ${rc1}  0
-    #Should Be Equal As Integers  ${rc2}  0
\ No newline at end of file
+    Run  echo '${yml}' > basic-compose.yml
+    ${rc}  ${output}=  Run And Return Rc And Output  DOCKER_HOST=${vch-ip}:2375 docker network create vic_default
```
Comment (caglar10ur) [inline tests/test-cases/Group3-Docker-Compose/3-3-Docker-Compose-Basic.robot]: do we need to do this? I was under the impression that compose creates a network if needed

**Comment by caglar10ur** [comment]
LGTM :sheep: :it:

File: tests/test-cases/Group3-Docker-Compose/3-3-Docker-Compose-Basic.robot (hunk 1 above)
Comment (sflxn) [inline tests/test-cases/Group3-Docker-Compose/3-3-Docker-Compose-Basic.robot]: It only creates a network if you do docker-compose run.  It's a weird situation where docker-compose create expects a pre-existing network and will error and spew out a python call stack.  Docker's been debating on whether to go ahead and create a default network for create for months with no decision yet.

**Comment by sflxn** [comment]
@cgtexmex Yes.  I was thinking we could wait till the true k/v store is implemented before persisting this data.  I didn't want to create a per container arbitrary k/v store for labels when it's going to get replace in a few months with the global k/v store.  What's your opinion?

**Comment by sflxn** [comment]
ping

**Pull request merged**